base/dcp_sharded.go

```diff
@@ -773,4 +779,21 @@ func (meh *sgMgrEventHandlers) OnFeedError(srcType string, r cbgt.Feed, feedErr
 		}
 	}
 
+	if len(indexUUID) == 0 {
```

Comment:
I know this code was lifted from cbft and I basically understand what it is doing, but I think some comments are worth mentioning.

Example comments: `indexUUUID = 0 means the feed is closing`

When is bucketName goign to be empty here? or manager is nil? 

nits for readability, take or leave: 

- I'd drop the nesting and do early returns here.
- I'd make `deleteIndexErr` as `err` so that we don't make future mistakes in logging since `err` is defined above in he function.

Reply:
Thanks for raising this - looking into this further I'm concerned that the in the case where the database is closed bucket still exists, the stats connections still aren't being closed.  Needs more investigation.